lib/modelHelpers.py

```diff
@@ -42,17 +112,36 @@ def make_prompt_from_editor_ctx(ctx: str):
     )
 
 
+def format_message(content: str):
+    message = {"type": "text", "text": content}
+    if content == "__IMAGE__":
+        clipped_image = clip.image()
+        if clipped_image:
+            data = clipped_image.encode().data()
+            base64_image = base64.b64encode(data).decode("utf-8")
+            message = {
+                "type": "image_url",
+                "image_url": {"url": f"data:image/;base64,{base64_image}"},
+            }
+    return message
+
+
 def generate_payload(
-    prompt: str, content: str, tools: Optional[list[Tool]] = None
+    prompt: str, content: str, tools: Optional[list[Tool]] = None, modifier: str = ""
 ) -> Tuple[Headers, Data]:
     """Generate the headers and data for the OpenAI API GPT request.
     Does not return the URL given the fact not all openai-compatible endpoints support new features like tools
     """
-    notify("GPT Task Started")
-
+    global stored_context
+    global thread_context
+    notification = "GPT Task"
```

Comment:
This is no longer going to say GPT task started I believe for normal requests

Reply:
I restored the original wording.